cogs/login.py

```diff
@@ -0,0 +1,146 @@
+            inline=True
+        )
+        embed.add_field(
+            name = ":yellow_circle: Peacock :yellow_circle:",
+            value = "u: " + config["Peacock"]["login"] + "\np: " + config["Peacock"]["password"], 
+            inline=True
+        )
+        embed.add_field(
+            name = ":nazar_amulet: Viki (KDramas) :nazar_amulet:",
+            value = "u: " + config["Viki"]["login"] + "\np: " + config["Viki"]["password"], 
+            inline=True
+        )
+        embed.add_field(
+            name = ":brown_circle: Showtime :brown_circle:",
+            value = "u: " + config["Showtime"]["login"] + "\np: " + config["Showtime"]["password"], 
+            inline=True
+        )
+
+        select = Select(                                        # Select Method that Creates Selection Menu for Embed
+            placeholder= "Choose a streaming service...",       # Placeholder Text
+            options=[                                           # Selection Options
+                discord.SelectOption(
+                    label="Crunchyroll", 
+                    emoji="🟠", 
+                    description="Crunchyroll Login"),
+                discord.SelectOption(
+                    label="Disney+", 
+                    emoji="🔵", 
+                    description="Disney+ Login"),
+                discord.SelectOption(
+                    label="HBO Max", 
+                    emoji="⚫", 
+                    description="HBO Max Login"),
+                discord.SelectOption(
+                    label="Hulu", 
+                    emoji="🟢", 
+                    description="Hulu Login"),          
+                discord.SelectOption(
+                    label="Netflix", 
+                    emoji="🔴", 
+                    description="Netflix Login"),
+                discord.SelectOption(
+                    label="NFL+", 
+                    emoji="🏈", 
+                    description="NFL+ Login"),
+                discord.SelectOption(
+                    label="Paramount+", 
+                    emoji="🗻", 
+                    description="Paramount+ Login"),
+                discord.SelectOption(
+                    label="Peacock", 
+                    emoji="🦚", 
+                    description="Peacock Login"),
+                discord.SelectOption(
+                    label="Viki", 
+                    emoji="🧿", 
+                    description="Viki Login"),
+                discord.SelectOption(
+                    label="Showtime", 
+                    emoji="🟤", 
+                    description="Showtime Login")           
+            ]
+        )
+
+        async def my_callback(interaction):
+            if interaction.user == ctx.author:      # Interaction Author Must be Original Sender
+                await interaction.response.send_message(config[select.values[0]]["password"])
```

Comment:
This is fine for now, but in the future we will want to make it more "secure". Ie either fix the problem we had where it wouldnt be copied to the clipboard or just dm the person with the info

Reply:
Will do it in a future update

Reply:
Resolved somewhat by doing `ephemeral=True` in interaction response.  Only author of interaction will see message.  Also, the bot will automatically delete the message after 15 seconds thanks to `delete_after=15`